pylint/lint/pylinter.py

```diff
@@ -1458,7 +1474,14 @@ def _add_one_message(
                 message_definition.msgid,
                 message_definition.symbol,
                 MessageLocationTuple(
-                    abspath, path, module or "", obj, line or 1, col_offset or 0
+                    abspath,
+                    path,
+                    module or "",
+                    obj,
+                    line or 1,
+                    col_offset or 0,
+                    end_lineno,
+                    end_col_offset,
```

Comment:
I was thinking again about this. Although I'm not 100% there yet, setting `end_lineno` and `end_col_offset` to `None` seems like the best we can do.

_We probably shouldn't touch `line or 1` and `col_offset or 0` though as this **will** break tools._